entity-framework/core/modeling/types-and-tables.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Types and Tables - EF Core
+author: rowanmiller
+ms.date: 10/27/2016
+ms.assetid: cbe6935e-2679-4b77-8914-a8d772240cf1
+uid: core/modeling/types-and-tables
+---
+# Types and Tables
```

Comment:
Mapping entity types to tables?

Reply:
I prefer to avoid tables in the name. Though @roji's point is right in the direction it is just not including/excluding types. Can we someone integrate that it is also about configuring facets over an entity type? Do we have other facets over entity types? Where are the docs for those stored? Perhaps comments/check constraints.

Reply:
-1 vote for Types & Tables. EntityType is better.

Reply:
@AndriySvyryd @smitpatel OK, I see your point.

How about Entity Types & Tables then? Am just trying to keep the title short (without Mapping), otherwise "Mapping entity types to tables" seems OK. I do think we should include "tables" in the title (after all the point of having entity types is that they are mapped to tables), and I agree with @smitpatel that we should consider adding additional facets over entity types in this page (comments/check constraints are good examples). I propose we do this later in a separate PR though.

There's the general question of provider-specific stuff, which I think belongs in the provider-specific documentation - but we could also link there from this page.

Reply:
EntityTypes can also map to views so tables in the title can mislead people even though they are looking for entity types only. Given relational nature of ToView, I believe it would also end up on this page.

> I propose we do this later in a separate PR though.

Of course. Also ToView. Though we should determine a good title according to future usage. We don't want to add more redirects when we can avoid.

Reply:
Down the line I could see a separate page for views (assuming one day we'll be able to create them from expressions or whatever), but I can also see them ending up here. We could call this something like "Entity Types, Tables and Views", that seems a bit long to me (but possible).

> We don't want to add more redirects when we can avoid.

Good point.

Reply:
Just EntityTypes?
EntityTypes and mapping?
Mapping entity types?

Reply:
Agreed on Entity Types.